features/cmake/find_package/exported_targets_multiconfig/README.md

```diff
@@ -0,0 +1,61 @@
+Using unobtrusive multi-config CMake with exported targets & find_package
+=========================================================================
+
+In this example
+---------------
+
+We are using CMake to declare, export and install the targets of a couple of libraries, and using Conan to manage
+the packages and using these libraries in a project. Everything is based on the ``find_package`` feature and working with 
+multiconfig packages (Release/Debug) to avoid installing the dependencies every time we change from Debug to Release in
+the Visual Studio IDE.
+
+ - We have a "hello" library consumed by a "bye" library and a project that uses the "bye" function.
+ - The "hello" library exports its own ``helloTargets.cmake`` files, everything is pure CMake, every time we call 
+   CMake with a different build type CMake exports a ``helloTargets-{build_type}.cmake`` script.
+ - The conan recipe uses the same binary package for any ``build_type``, by removing the ``build_type`` setting.
```

Comment:
I am confused, I thought the goal of this generator was to be able to consume single-config packages for debug and release, not to consume multi-config packages. You are creating multi-config packages here.

Reply:
I see now there is another section below, that handle that case. Good.
But I think I will promote the single-config packages as the first, primary example, because that is the default for Conan. Then I would put the multi-config packages last.

Also, I need the titles should be improved for clarifying about single-config and multi-config

Reply:
The folder is called `exported_targets_multiconfig` and the pull request `cmake_find_package_multi`. This example, IS NOT using the new generator